Services/TagService.cs

```diff
@@ -0,0 +1,37 @@
+using Neo4jClient;
+using TAG.Nodes;
+using TAG.Extensions;
+using TAG.Constants;
+using TAG.DTOS;
+using TAG.Services.Interfaces;
+
+namespace TAG.Services
+{
+    public class TagService : ITagService
+    {
+        private readonly IGraphClient _graphClient;
+
+        public TagService(IGraphClient graphClient)
+        {
+            _graphClient = graphClient;
+        }
+
+        public async Task<TagSearchResponseDTO> SearchTagsAsync(TagSearchRequestDTO request)
+        {
+            var query = _graphClient.Cypher
+                .Match("(tag:Tag)")
```

Comment:
Please add Tag to Constants/NodeNames and use it when creating a query

Reply:
fixed